### ISSUE TITLE
Filtering "action_mqtt_payload": "[detections]"

### ISSUE THREAD

**Issue by bmorgan99** [VorlonCD/bi-aidetection#156]
I am using the mqtt detection payload to pop up the hits in a small Home Assistant card.  I'd like to have just have the relevant, high quality stuff instead of everything.  Forgive me if this is already possible and my mojo is just lacking.

**Comment by VorlonCD** [comment]
History tab > Settings > enable "Show only relevant objects".    But why wouldnt you want to see potted plants and skateboards detected?  :)

**Comment by bmorgan99** [comment]
That, I had found and works in the AI Tool app.  I'm comparing to an earlier version where I only saw "Person, Person".  Is it the same data going to mqtt?  Maybe I was very fortunate before and was never looking when chairs were on the prowl.

**Comment by VorlonCD** [comment]
PSA: Chairs and skateboards can be very dangerous.  Mostly when Whiskey is involved but other times too I believe.

With this new version, go into Camera > Actions and you can control how each individual detection is displayed.  So you can reduce the detail if you always want it to be just PERSON you can set it to **[label]** only.

[AITool-VorlonCD-v2.0.626.zip](https://github.com/VorlonCD/bi-aidetection/files/5898931/AITool-VorlonCD-v2.0.626.zip)

**Comment by balucanb** [comment]
Ok @VorlonCD you gonna need to to dumb it down a bit so I don't choke on my tendies... are you describing something that only pertains to MQTT? You said control each detection, as you can see mine says person 73% but I am detecting several items on this camera and I have no idea where the 73% is coming from since my confidence levels (on the camera page) are 50-100? or do you mean if you want ONLY to detect person remove everything except [label] and I assume you can adjust confidence (on the ACTIONS page) to what you want (overriding settings on the camera page) and if you do not change it you have made the default 73% for person? Confused yet, sorry LOL I tried to be clear.

![Capture](https://user-images.githubusercontent.com/34525580/106371230-8b7c8500-6327-11eb-8f46-537b200f49bd.PNG)

**Comment by bmorgan99** [comment]
Perfect.  Thanks

**Comment by VorlonCD** [comment]
@balucanb - Anywhere you see a description of a detected object (IE 'PERSON 73%') is defined by "**[detection] format**" setting.   Then, for Telegram, MQTT, etc you can use **[detection]**, **[detections]** or **[summary]** variables to send the detections in that format to the service. 

Where you see 'Person (73%)' it is just an EXAMPLE.   Although if the camera within that session has had a recent detection I try to use the last real detection for the display.
